### ISSUE TITLE
Get-Package does not support "DownloadCount" and "Authors"

### ISSUE THREAD

**Issue by alpaix** [NuGet/Home#3110]
Reported in support request #SR-411 by [samdevx1029](http://www.nuget.org/Profiles/samdevx1029)

> The latest NuGet version does not support "DownloadCount" and "Authors" using its PowerShell module. The following used to work:

``` powershell
Get-Package -Filter moq -ListAvailable | Select-Object Id, Version, Description, DownloadCount, Authors 
```

> But now, DownloadCount and Authors show empty! Do you know why this is happening?

**Comment by rrelyea** [comment]
@joelverhagen investigated similar issues. Same basic problem?

**Comment by joelverhagen** [comment]
Yes, the only fields available right now are ID, description, version, and license URL.

**Comment by harikmenon** [comment]
can they use the V2 feed to get this data?

**Comment by joelverhagen** [comment]
A cursory look at the code indicates that the internal model has a lot more information than ID, description, version, and license URL. These two requested fields are available and can be plumbed through to PMC.

**Comment by harikmenon** [comment]
Okay. then we should probably fix this. Do you we know when we regressed this?

**Comment by samdevx1029** [comment]
Hello, any update on this?

**Comment by jafin** [comment]
Similar Find-Package does not return download count.

``
Find-Package dapper -First 100 | Select-Object ID, Version, DownloadCount, Description | Sort-Object -Descending -Property DownloadCount
``

**Comment by jez9999** [comment]
Amazing that this hasn't been fixed yet.
